// The try statement defines a code block to run (to try).

// The catch statement defines a code block to handle any error.

// The finally statement defines a code block to run regardless of the result.

// The throw statement defines a custom error.
try {
  consol.log("hello This error is generated by me");
} catch (error) {
  console.log("error", error.message);
}

// start from eval error
